src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.js

```diff
@@ -395,25 +400,29 @@ const ImageAnnotationModalMap = ({
       'bottom-left': topRight,
       'bottom-right': topLeft,
     }
-
-    setSelectedPoint({
+    const selectedPointToUse = {
       id: properties.id,
       popupAnchorLngLat: latLngLookupByAnchorPosition[popupAnchorPosition],
       popupAnchorPosition,
       bounds,
-    })
+    }
+    setSelectedPoint(selectedPointToUse)
+
+    return selectedPointToUse
```

Comment:
Returning the selected point info so we dont have a infinite loop in a useEffect that reads and sets the same value. WE use `popupAnchorLngLat` to center the map